x-pack/legacy/plugins/lens/public/indexpattern_plugin/dimension_panel/dimension_panel.tsx

```diff
@@ -133,24 +151,38 @@ export const IndexPatternDimensionPanel = memo(function IndexPatternDimensionPan
           const newColumn = hasFieldChanged
             ? changeField(selectedColumn, currentIndexPattern, droppedItem.field)
             : buildColumn({
```

Comment:
This won't carry over settings made to the moved column - e.g. if the label or `size` is changed in a terms operation - it probably makes sense to include it.

Reply:
That's a good point. It's possible that the field could be accepted as a drop target, but the operation / current configuration wouldn't be. (e.g. dragging "Sum of bytes" to the X axis would switch it to "top values of bytes" or something). Or maybe we should not allow the drop if the current operation isn't compatible with the target?